curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md

```diff
@@ -7,7 +7,9 @@ dashedName: step-13
 
 # --description--
 
-Give your `.bb1` element these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`. This will center the parts of the building using "flex" or "flexbox". You will learn about it in more detail on another project.
+1. Turn your `.bb1` element into a flexbox parent so you can center the parts of the building.<br>- hint: You should give `.bb1` a `display` of `flex`
```

Comment:
The `hints` are what go in the next section to display when a test fail.

It seems you misinterpreted what was intended with "split in 3 different steps"

please check our documentation and look at how the project works in the curriculum.